Source/Private/ASDisplayNodeInternal.h

```diff
@@ -319,4 +326,10 @@ FOUNDATION_EXPORT NSString * const ASRenderingEngineDidDisplayNodesScheduledBefo
 
 @end
 
+@interface ASDisplayNode (InternalPropertyBridge)
+
+@property (nonatomic, assign) CGFloat layerCornerRadius;
```

Comment:
May be worth dropping a comment on this property explaining its distinction from `.cornerRadius`? Seems obvious to us today but who knows tomorrow